tests/import_specifications/test_individual_parsers.py

```diff
@@ -239,31 +238,31 @@ def test_xsv_parse_fail_unequal_rows(temp_dir: Path):
         "head1, head2, head3\n",
         "Head 1, Head 2\n",
     ]
-    _xsv_parse_fail(temp_dir, lines, parse_csv, err)
+    _xsv_parse_fail(temp_dir, lines, parse_csv, err, ErrorType.INCORRECT_COLUMN_COUNT)
```

Comment:
Is it expected to be case-sensitive?
Would:
```
data type: foo; columns: 3; version: 1\n
... etc
```
pass or fail?

Reply:
Not really applicable for this PR, just curious.

Reply:
It's case sensitive - it's just a simple regex and I didn't bother to make it case insensitive, because the headers aren't supposed to be touched by the user, as noted in the first troubleshooting line of the [readme](https://github.com/kbase/staging_service/pull/142).